src/components/VideoPlayer/index.js

```diff
@@ -0,0 +1,203 @@
+    }
+
+    this.startSecondsTimer()
+  }
+
+  handleVideoEnd = () => {
+    this.currentTime = 0
+    this.hasEnded = true
+    if (this.props.isLooped) {
+      this.video.play()
+    } else if (this.props.endscreenComponent) {
+      this.setState({
+        endscreenOpen: true,
+      })
+    }
+    if (this.props.onEnd) {
+      this.props.onEnd(this.video)
+    }
+  }
+
+  handleReplayClick = () => {
+    this.video.play()
+    this.setState({
+      endscreenOpen: false,
+    })
+  }
+
+  replaceWith = (videoId) => {
+    if (this.video.customOverlay) {
+      this.video.customOverlay.close()
+    }
+
+    this.video.catalog.getVideo(videoId, (error, video) => {
+      if (error && this.props.onError) {
+        this.props.onError(error, this.video)
+        return
+      }
+      this.video.catalog.load(video)
+      this.hasEnded = false
+      this.currentTime = 0
+      this.setState({
+        endscreenOpen: false,
+      })
+      this.video.play()
+    })
+  }
+
+  startSecondsTimer = () => {
+    if (this.props.onTimeChange) {
+      this.video.on('timeupdate', () => {
+        const currentTime = Math.floor(this.video.currentTime())
+        if (this.currentTime < currentTime) {
+          this.currentTime = currentTime
+          this.props.onTimeChange(currentTime)
+        }
+      })
+    }
+  }
+
+  renderEndScreen = () => (
+      <div className='bc-player-endscreen'>
+        <div
+          className='bc-player-endscreen__replay'
+          onClick={this.handleReplayClick}
+        >
+          <svg
+            height='48'
+            viewBox='0 0 48 48'
+            width='48'
+            xmlns='http://www.w3.org/2000/svg'
+          >
+            <path
+              d='M0 0h48v48H0z'
+              fill='none'
+            />
+            <path
+              fill='#fff'
+              d='M24 10V2L14 12l10 10v-8c6.63 0 12 5.37 12 12s-5.37 12-12 12-12-5.37-12-12H8c0 8.84 7.16 16 16 16s16-7.16 16-16-7.16-16-16-16z'
+            />
+          </svg>
+        </div>
+        <div className='bc-player-endscreen__content'>
+          {this.props.endscreenComponent}
+        </div>
+      </div>
+  )
+
+  render () {
+    return (
+      <div
+        className={cn('bc-player', {
+          'bc-player--endscreen-open': this.state.endscreenOpen,
+          'bc-player--has-breakpoints': this.props.hasBreakpoints,
+        })}
+      >
+        {this.props.endscreenComponent && this.renderEndScreen()}
+        <div className='bc-player__wrapper'>
+          <video
+            data-application-id
+            ref={(ref) => {
+              this.playerRef = ref
+            }}
+            className={cn(
+              'video-js',
+              'bc-player__video',
+              `bc-player__video--${this.props.theme}`,
+            )}
+            data-embed='default'
+            data-video-id={this.props.videoId}
+            data-player-id={this.props.playerId}
+            data-account='5344802162001'
```

Comment:
You also used this value in this [line](https://github.com/yankaindustries/mc-components/pull/43/files#diff-c00393e43603fb0a9c25af3001fe7719R8), what do you think about move it to a constant?

Reply:
Those are actually different numbers, they looks super similar though